dockerfiles/run/guice/destination/conf/elasticsearch.properties

```diff
@@ -26,3 +26,8 @@ elasticsearch.nb.shards=1
 elasticsearch.nb.replica=0
 elasticsearch.retryConnection.maxRetries=7
 elasticsearch.retryConnection.minDelay=3000
+
+# This properties defines the maximal length of search results. Higher values
+# means more results but costs more performances.
+# Default value is 100
+elasticsearch.search.size=100
```

Comment:
Oh I just ask the question : maybe you want me to implement this using a scroll ?